oper8/watch_manager/ansible_watch_manager/modules/k8s_application.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
```

Comment:
I'm curious why this change was added. Does the new `ansible` base image not have `/usr/bin/env` in it? In general, that's the preferred method for a shebang pointing at the generic version of `python` or `bash` in a script like this.

Reply:
The image has `/usr/bin/env` and it works if I run manually in the container. I also think it's usually right way. However, ansible raise error as `/usr/bin/env python` not found. It is currently not allowed in ansible according to [this issue comment](https://github.com/ansible/ansible/issues/83603).